docs/src/components/SidebarSection.js

```diff
@@ -10,8 +10,8 @@ export default function SidebarSection({
   section: sidebarIndexType,
 |}): Node {
   return (
-    <>
-      <Box padding={2} marginTop={4} role="list">
+    <Box role="list">
```

Comment:
A11Y Fix: we previous had a `<Box role="list" />` without any `role="listitem"` in them.